test/tests/functional/pbs_job_dependency.py

```diff
@@ -353,3 +353,182 @@ def test_dependency_on_multiple_finished_job(self):
 
         a = {ATTR_depend: 'beforenotok:' + j1 + ":" + j2}
         self.check_job(a, reject_msg, 'F')
+
```

Comment:
I just noticed on line 83 of this test there is an over-indent.  If you don't mind, could you correct it?
```
            for job_list in check_dl:
                     self.assertIn(job, job_list)
```